internal/controllers/analyzer/analyzer_test.go

```diff
@@ -359,13 +346,11 @@ func TestAnalyze(t *testing.T) {
 
 		horusecAPI := testutil.NewHorusecAPIMock()
 		horusecAPI.On("SendAnalysis").Return(nil)
-		horusecAPI.On("GetAnalysis").Return(new(entitiesAnalysis.Analysis), nil)
-
-		docker := docker.New(testutil.NewDockerClientMock(), cfg, uuid.New())
+		horusecAPI.On("GetAnalysis").Return(new(analysis.Analysis), nil)
 
-		analysis := new(entitiesAnalysis.Analysis)
-		analysis.AnalysisVulnerabilities = append(
-			analysis.AnalysisVulnerabilities, entitiesAnalysis.AnalysisVulnerabilities{
+		analysiss := new(analysis.Analysis)
```

Comment:
Typo in `analysiss`

Reply:
Is to avoid variable name with the same import.